test/gosec.spawn.test.js

```diff
@@ -1,36 +1,41 @@
 // Copyright 2018 VMware, Inc.
 // SPDX-License-Identifier: BSD-2-Clause
 
-const gosec = require('../gosec/gosec')
 const fs = require('fs-extra')
 
-describe('Spawn gosec tests', () => {
-  test('Empty input for gosec', async () => {
-    const gosecResult = await gosec('', [])
-    expect(gosecResult).toBeFalsy()
-  })
+const { run } = require('../runner')
+const Gosec = require('../linters/gosec')
+
+function gosec (dir, files) {
+  return run(new Gosec(), dir, files)
+}
+
+describe('Gosec runner', () => {
+  test('Finds issues in invalid file', async () => {
```

Comment:
I don't think that "Finds issues in invalid file" is a good name for that test.
Because when you say invalid for me it means a file with non valid Go code which isn't compiling.

Reply:
I'll change the title

Reply:
What title did you have in mind?

Reply:
"Find issues in non secure file" or
"Find issues in vulnerable file" or
"Find issues in problematic file"
If change that maybe you should also rename the mix.invalid file.

Reply:
Switched to 'vulnerable'.